eng/helix/helix.proj

```diff
@@ -23,13 +23,30 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>$(NETCoreSdkVersion)</DotNetCliVersion>    
+    <DotNetCliChannel>Current</DotNetCliChannel>
 
     <!-- Copied from Microsoft.NET.DefaultOutputPaths.targets in the .NET SDK. The Helix SDK contains nothing similar. -->
     <BaseOutputPath Condition="'$(BaseOutputPath)' == ''">bin\</BaseOutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' == 'AnyCPU'">$(BaseOutputPath)$(Configuration)\</OutputPath>
     <OutputPath Condition="'$(OutputPath)' == '' and '$(PlatformName)' != 'AnyCPU'">$(BaseOutputPath)$(PlatformName)\$(Configuration)\</OutputPath>
   </PropertyGroup>
 
+  <!-- Specify the runtime we need which will be included as a correlation payload -->
+  <ItemGroup>
+    <!--
+      When the targeting pack builds, it has exactly the same version as the shared framework. Passing
+      SharedFxVersion because that's needed even when the targeting pack isn't building.
```

Comment:
What do these comments mean❔

Reply:
I just copied these from helix.targets which currently uses this version for our helix jobs, I have no idea :)

Reply:
The comments are about Microsoft.AspNetCore.App in that context. They don't apply here.